pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -2715,3 +2718,149 @@ func TestBuildAutoMtlsSettings(t *testing.T) {
 		})
 	}
 }
+
+func TestApplyDestinationRuleOSCACert(t *testing.T) {
+	features.VerifyCertAtClient = true
```

Comment:
to be complete the test case should test the feature disable behavior as well, test case can have this as part of the dimension.

```golang
{
  enableVerifyCertAtClient: true/false,
  drContainsCaCertificate: true/false
}
```